calico_rkt/calico_policy.py

```diff
@@ -0,0 +1,199 @@
+# Copyright 2015 Metaswitch Networks
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from constants import *
+from calico_rkt import configure_logging
+from pycalico.datastore import DatastoreClient
+from pycalico.datastore_datatypes import Rule, Rules
+
+KUBERNETES_DEFAULT_PROFILE = "default_profile"
+
+_log = logging.getLogger(__name__)
+configure_logging(_log)
+
+class BasePolicyPlugin(object):
+    def __init__(self, network_config, env):
```

Comment:
There is a whole bunch of stuff here that isn't used, and so doesn't need to be stored.

I don't think we should be copying the __init__ procedure from the CniPlugin class, since they're pretty different.  It is OK for the subclasses of BasePolicyPlugin to require different arguments in __init__, and it is up to them to decide what the required arguments are.

For example:
```
# DefaultPolicyDriver only needs the network name.
driver = DefaultPolicyDriver(network_name)
driver.apply_policy(endpoint)

# KubernetesDefaultDriver doesn't need any extra arguments
driver = KubernetesDefaultDriver()
driver.apply_policy()

# KubernetesAdvPolicyDriver needs more information!
driver = KubernetesAvcPolicyDriver(env, calico_config)
driver.apply_policy()
```